docs/docs/examples-tutorials/recipes/running-playwright-performance-tests-with-artillery-and-tracetest.mdx

```diff
@@ -12,6 +12,12 @@ keywords:
   - artillery
   - load testing
   - performance testing
+  - e2e testing
+  - playwright
+  - end to end testing
+  - end-to-end testing
```

Comment:
This in the list twice?

Reply:
Yeah that's fine. Semrush thinks theyre two distinct keywords.